db/migrate/20160921214741_create_sodas.rb

```diff
@@ -0,0 +1,20 @@
+class CreateSodas < ActiveRecord::Migration[5.0]
+  def change
+    create_table :sodas do |t|
+      t.references :user, foreign_key: true
+      t.references :country, foreign_key: true
+      t.string :name
+      t.string :brand
+      t.string :origin_data
+      t.boolean :is_diet
+      t.boolean :is_caffeinated
+      t.string :color
+      t.references :users, column: :creator_id
```

Comment:
You're 100% right - that does not work as coded. I finally found the doc/code to help:

http://edgeapi.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_foreign_key

https://github.com/rails/rails/blob/v5.0.0.rc2/activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb#L590

Turns out these t.references are shorthand helpers for the schema statement helpers, and their options and use aren't fully explained in the guides. Should work on next pr